test/ui-test/testSuites/suite_status/tst_ChatFlow/test.feature

```diff
@@ -83,3 +83,10 @@ Feature: Status Desktop Chat
          When user sends the emoji sunglasses with message wow I'm so cool
          Then the emoji 😎 is displayed in the last message
          And the message wow I'm so cool is displayed in the last message
+
+     Scenario: User can type message with emoji autoreplace
+         When user joins chat room automation-test
+         Then user is able to send chat message (without verification)
+           | message                |
+           | Hello :)               |
+         Then the emoji 🙂 is displayed in the last message
```

Comment:
Similar comment here than @jrainville . Could it make sense to add a general message (if we still we don't have it) like:
`the last message displayed is XXX`